go-controller/pkg/ovn/base_network_controller_pods.go

```diff
@@ -444,7 +446,7 @@ func (bnc *BaseNetworkController) ensurePodAnnotation(pod *kapi.Pod, nadName str
 }
 
 func (bnc *BaseNetworkController) addLogicalPortToNetwork(pod *kapi.Pod, nadName string,
-	network *nadapi.NetworkSelectionElement) (ops []ovsdb.Operation,
+	network *nadapi.NetworkSelectionElement, lspEnabled *bool) (ops []ovsdb.Operation,
```

Comment:
curious: are we ever using this method to transition an existing LSP from `enabled==true`to `enabled==nil`?

nit: probably `enable` vs `lspEnable` as the method name is already referring to the port.

Reply:
>curious: are we ever using this method to transition an existing LSP from `enabled==true` to `enabled==nil`?

no. prior to this PR this field was not updated, and now we only set to true/false.

>nit: probably enable vs lspEnable as the method name is already referring to the port.

DONE